### PR TITLE
Add non-async chunk support for Qwen3-TTS

### PR DESCRIPTION
**Pull request by linyueqian**
## Summary
- Add `talker2code2wav` non-async stage input processor for Qwen3-TTS
- Add `qwen3_tts_no_async_chunk.yaml` stage config
- Add benchmark scripts to compare async_chunk on vs off (TTFP, E2E, RTF)

## How to test
```bash
# Serve with non-async chunk config
vllm-omni serve <model> \
    --stage-configs-path vllm_omni/model_executor/stage_configs/qwen3_tts_no_async_chunk.yaml \
    --trust-remote-code --omni

# Test with curl
curl http://localhost:8000/v1/audio/speech \
    -H "Content-Type: application/json" \
    -d '{"input": "Hello world", "voice": "Vivian", "response_format": "wav"}' \
    -o test.wav

# Run full benchmark (async on vs off)
GPU_DEVICE=0 MODEL=<model> bash benchmarks/qwen3-tts/vllm_omni/run_async_chunk_benchmark.sh
```

cc @Sy0307 can you test this on your machine?

### CODE REVIEW

File: vllm_omni/model_executor/stage_configs/qwen3_tts_no_async_chunk.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+async_chunk: false
```
Comment (hsliuustc0106) [inline vllm_omni/model_executor/stage_configs/qwen3_tts_no_async_chunk.yaml:1]: is this called default args @lishunyang12

**Comment by Sy0307** [comment]
> cc @Sy0307 can you test this on your machine?

I will test it later. Thanks for your contributing.

**Comment by Sy0307** [comment]
Test results on 5090:

| Metric | async_chunk ON | async_chunk OFF |
|--------|---------------|-----------------|
| TTFP mean | 1062.4 ms | 1065.9 ms |
| TTFP median | 1062.5 ms | 1068.5 ms |
| E2E mean | 3001.2 ms | 2945.2 ms |
| E2E median | 2992.4 ms | 2991.3 ms |
| E2E p90 | 3777.4 ms | 3500.1 ms |
| RTF mean | 0.523 | 0.524 |
| Audio throughput | 1.91 s/s | 1.91 s/s |

**Comment by hsliuustc0106** [comment]
I think we need to add test for this model in different modes

**Comment by linyueqian** [comment]
Added nightly e2e tests for the non-async-chunk mode (English + Chinese) in `tests/e2e/online_serving/test_qwen3_tts.py`, with a corresponding Buildkite step in `test-nightly.yml`.

File: vllm_omni/model_executor/stage_input_processors/qwen3_tts.py [hunk 2]
```diff
@@ -5,6 +5,37 @@
...
+        # Filter zero-padded frames (EOS/invalid steps), matching _extract_last_frame behavior
+        valid_mask = audio_codes.any(dim=1)
+        audio_codes = audio_codes[valid_mask]
+        # Code2Wav expects codebook-major flat: [Q*num_frames]
+        codec_codes = audio_codes.transpose(0, 1).cpu().reshape(-1).tolist()
+        code2wav_inputs.append(
+            OmniTokensPrompt(
+                prompt_token_ids=codec_codes,
+                multi_modal_data=None,
+                mm_processor_kwargs=None,
```
Comment (hsliuustc0106) [inline vllm_omni/model_executor/stage_input_processors/qwen3_tts.py:33]: The zero-padding filter logic is critical for correctness but has no unit test. Add a test case verifying that: (1) all-zero frames are filtered, (2) valid frames are preserved, (3) edge cases like all-zero or no-zero input work correctly.

File: benchmarks/qwen3-tts/vllm_omni/run_async_chunk_benchmark.sh [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    --output "${RESULT_DIR}/qwen3_tts_async_chunk_all.png"
+
+echo ""
+echo "============================================================"
+echo " Benchmark complete!"
+echo " Results: ${RESULT_DIR}/"
+echo " Plots:"
+echo "   - ${RESULT_DIR}/qwen3_tts_async_chunk_ttfp.png"
+echo "   - ${RESULT_DIR}/qwen3_tts_async_chunk_all.png"
+echo "============================================================"
```
Comment (hsliuustc0106) [inline benchmarks/qwen3-tts/vllm_omni/run_async_chunk_benchmark.sh:167]: The benchmark script is excellent, but the PR description should include actual benchmark results. Run the script once and paste the comparison table or link to a plot showing TTFP/E2E/RTF improvements.

File: vllm_omni/model_executor/stage_configs/qwen3_tts_no_async_chunk.yaml (hunk 1 above)
Comment (hsliuustc0106) [inline vllm_omni/model_executor/stage_configs/qwen3_tts_no_async_chunk.yaml:1]: Add a comment at the top explaining when to use this config vs the default async_chunk config. Example: "Use this config when streaming is not required and you want simpler processing. Disables chunk-level streaming for lower complexity but potentially higher latency."

File: tests/e2e/online_serving/test_qwen3_tts.py [hunk 4]
```diff
@@ -225,3 +223,63 @@ def test_models_endpoint(self, omni_server) -> None:
...
+        [
+            "--stage-configs-path",
+            stage_config_path,
+            "--stage-init-timeout",
+            "120",
+            "--trust-remote-code",
+            "--enforce-eager",
+            "--disable-log-stats",
+        ],
+    ) as server:
```
Comment (hsliuustc0106) [inline tests/e2e/online_serving/test_qwen3_tts.py:244]: The new test class duplicates setup code from the async_chunk tests. Consider extracting common fixtures or using pytest.mark.parametrize to reduce duplication.
